ReactAndroid/README.md

```diff
@@ -12,9 +12,9 @@ Assuming you have the [Android SDK](https://developer.android.com/sdk/installing
 
 Make sure you have the following installed:
 
-- Android SDK version 22 (compileSdkVersion in [`build.gradle`](build.gradle))
-- SDK build tools version 22.0.1 (buildToolsVersion in [`build.gradle`](build.gradle))
-- Android Support Repository 17 (for Android Support Library)
+- Android SDK version 23 (compileSdkVersion in [`build.gradle`](build.gradle))
+- SDK build tools version 23.0.1 (buildToolsVersion in [`build.gradle`](build.gradle))
+- Android Support Repository 20 (for Android Support Library)
```

Comment:
Just checked 20 is the latest one now. We'll have to keep the version up-to-date which is tedious and the version doesn't need to be strictly 20 (it builds fine with 17).

Can you say e.g. >= 17 or remove the version?